docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "3.8"
+
+name: titan-node
+
+services:
+  titan-node-pool:
+    build: .
+    image: titan-node-pool
+    container_name: titan-node-pool
+    restart: unless-stopped
+    runtime: nvidia
+    command:
```

Comment:
This could be improved by supporting `yml` or `json` based config files like is done in the LivePeer client:

```suggestion
    command: ["-config", "<PATH_TO_CONFIG>"]
```